sources/packages/api/src/sfas-integration/sfas-files/sfas-part-time-application-record.ts

```diff
@@ -0,0 +1,47 @@
+import { parseDate } from "./sfas-parse-utils";
+import { SFASRecordIdentification } from "./sfas-record-identification";
+
+/**
+ * This record contain data related to an Student part time Application on SFAS.
+ */
+export class SFASPartTimeApplicationRecord extends SFASRecordIdentification {
+  constructor(line: string) {
+    super(line);
+  }
+  /**
+   * The unique key/number used in SFAS to identify this individual (Sail_extract_data.individual_idx).
```

Comment:
Please be consistent on the 'Sail_extract_data' in the comments of the model

Reply:
commented above